packages/backend-function/src/factory.ts

```diff
@@ -121,6 +123,20 @@ export type FunctionProps = {
    * schedule: "0 9 * * 2" // every Monday at 9am
    */
   schedule?: FunctionSchedule | FunctionSchedule[];
+
+  /**
+   * Attach Lambda layers to a function
+   * @example
+   * import { referenceFunctionLayer } from "@aws-amplify/backend";
+   *
+   * layers: {
+   *    myModule: referenceFunctionLayer("arn:aws:lambda:<current-region>:<account-id>:layer:myLayer:1")
+   * },
```

Comment:
does this render okay? I remember some wonkiness with multiline examples and spacing. Also can we use the powertools example?

Reply:
![image](https://github.com/user-attachments/assets/f94e7491-de6a-4cdc-8c6c-b146333f6ad0)

it appears to render it correctly